src/generators/building-rollup-ts/templates/package.json

```diff
@@ -1,12 +1,18 @@
 {
   "scripts": {
     "build": "rimraf dist && tsc && rollup -c rollup.config.js",
-    "start:build": "npm run build && web-dev-server --root-dir dist --app-index index.html --open --compatibility none"
+    "start:build": "web-dev-server --root-dir dist --app-index index.html --open"
   },
   "devDependencies": {
-    "@open-wc/building-rollup": "^1.10.0",
-    "deepmerge": "^4.2.2",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.51.2"
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@web/rollup-plugin-html": "^1.8.0",
+    "@web/rollup-plugin-import-meta-assets": "^1.0.7",
+    "@babel/preset-env": "^7.14.1",
```

Comment:
Do we care about alphabetizing these?

Reply:
I added a step for that after merging all the package.json